service-loadbalancer/template.cfg

```diff
@@ -54,9 +54,17 @@ defaults
     # Maximum allowed time to wait for a new HTTP request to appear.
     timeout http-keep-alive 60s
 
+    # default traffic mode is http
+    mode http
+
+    # default default_backend. This allows custom default_backend in frontends
```

Comment:
please clarify this comment by saying mode is overwritten in the tcp segment below